platforms/CCStudio/CppUTestExtRunAllTests1.pjt

```diff
@@ -38,7 +38,6 @@ Source="..\..\src\CppUTestExt\MockExpectedCallsList.cpp"
 Source="..\..\src\CppUTestExt\MockFailure.cpp"
 Source="..\..\src\CppUTestExt\MockNamedValue.cpp"
 Source="..\..\src\CppUTestExt\MockSupport.cpp"
-Source="..\..\src\CppUTestExt\MockSupport_c.cpp"
```

Comment:
This file was still in there although unneeded (MockSupport_c is tested in CppUTestExtRunAllTests2.pjt). I removed it here since we are coming up against memory limitations.